app/src/db/MissedPingsMq.js

```diff
@@ -40,6 +42,23 @@ const MissedPingsMq = {
     console.log(job);
   },
 
+  async populateStartSoloQueues() {
```

Comment:
If you add `await this.boss.deleteAllQueues();` to this method then it will clear out all old queues and jobs from the database. I originally had this in `init` in some of my branches (I don't think it made its way to main yet) but it seems like it belongs better here now.

Reply:
That's actually a pretty neat method. Cool, I'll add that in 👍